erpnext/accounts/doctype/payment_entry/payment_entry.py

```diff
@@ -1522,6 +1523,8 @@ def get_payment_entry(dt, dn, party_amount=None, bank_account=None, bank_amount=
 	pe.paid_amount = paid_amount
 	pe.received_amount = received_amount
 	pe.letter_head = doc.get("letter_head")
+	if dt == 'Purchase Order':
```

Comment:
This will also be applicable for other transactions like "Sales Order", "Purchase Receipt", "Delivery Note", "Sales Invoice" and "Purchase Invoice". Can we make this more generic to handle for all doctypes?

Reply:
We can avoid 'Purchase Receipt' and 'Delivery Note', as there are no process available to create Payment Entry from them.